### ISSUE TITLE
Invalid name error in FindNeighbors 

### ISSUE THREAD

**Issue by iiiir** [satijalab/seurat#3154]
I have been sstuck by this error message. 
I thought it is something related to the cell names, and tried several ways to correct them (e.g. remove :, space, etc).
Can someone shed any insight?

![image](https://user-images.githubusercontent.com/4137873/84813536-f65d9380-afc4-11ea-924f-e3b542c8d8fa.png)

Thanks,
Shuoguo

**Comment by iiiir** [comment]
It seems have nothing to do with the cell names as changing cell names does not help
```
colnames(data.raw) <- paste0("test-",seq(1, dim(data.raw)[2] ))
myobj <- CreateSeuratObject(counts = data.raw, project = "ABIO")
...
```
Also gene names seems fine as these two commands does not show anything suspecious:
```
head(sort(rownames(myobj)))
tail(sort(rownames(myobj)))
```


Thanks,

**Comment by iiiir** [comment]
Shrink the matrix would work...
![image](https://user-images.githubusercontent.com/4137873/84827329-2878f080-afd9-11ea-9680-e92dfa8f9bd0.png)

**Comment by iiiir** [comment]
**update:** 
this issue only affects seurat version 3.0.1
updated to seurat 3.1.5 and the issue resolved.

**Comment by timoast** [comment]
I will close this now if it's fixed in 3.1.5
